docs/kratos/bring-your-own-ui/10_custom-ui-advanced-integration.mdx

```diff
@@ -99,6 +99,31 @@ the flow data across two forms.
 </form>
 ```
 
+#### Upstream provider parameters
+
+In some cases, it is necessary to pass additional parameters to the upstream provider. For example, when using the Google
```

Comment:
```suggestion
In some cases, it is possible to pass additional parameters to the upstream provider. For example, when using the Google
```
Presumably, passing these Google parameters is optional? In which case I think it's clearer to say that it is "possible" rather than "necessary" to pass these parameters.